apps/common/lib/lexical/ast/callable.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Lexical.Ast.Callable do
```

Comment:
you said there were problems with indexing these things, why not fix them in the indexer rather than here?